ui/desktop/electron-app/src/services/cache-daemon-manager.js

```diff
@@ -117,9 +117,18 @@ class CacheDaemonManager {
     return generateErrorPromise(stderr);
   }
 
-  search(requestData) {
+  async search(requestData) {
+    const start = Date.now();
+    // Log request data, but clean up token from log
+    const { auth_token_id, token, ...logRequestData } = requestData;
+
     if (isWindows()) {
-      return searchCliCommand(requestData);
+      const result = await searchCliCommand(requestData);
+
+      const end = Date.now();
+      log.debug(`Search request took ${end - start} ms`, logRequestData);
+
+      return result;
     }
 
     delete requestData.token;
```

Comment:
line 123 is already deleting token, isn't?

Reply:
Yes it is but it doesn't get deleted for the windows search so I just did it again, otherwise i'd have to do two separate statements